test/sv/interfaces.mlir

```diff
@@ -35,6 +35,16 @@ func @test() {
   // CHECK-NEXT:   sv.interface.modport @dataflow_out ("output" @data, "output" @valid, "input" @ready)
   // CHECK-NEXT: }
 
-  // CHECK-NEXT: return
-  return
+  rtl.externmodule @Rcvr (%m: !sv.interface.modport<@handshake_example::@dataflow_in>)
+  // CHECK-LABEL: rtl.externmodule @Rcvr(!sv.interface.modport<@handshake_example::@dataflow_in> {rtl.name = "m"})
+
+  rtl.module @Top () {
+    %iface = rtl.wire : !sv.interface<@handshake_example>
+    %ifaceInPort = sv.extract @dataflow_in %iface : !sv.interface<@handshake_example>
+    rtl.instance "rcvr" @Rcvr(%ifaceInPort) : (!sv.interface.modport<@handshake_example::@dataflow_in>) -> ()
+  }
+  // CHECK-LABEL: rtl.module @Top() {
+  // CHECK-NEXT:    %iface = rtl.wire : !sv.interface<@handshake_example>
+  // CHECK-NEXT:    %0 = sv.extract @dataflow_in %iface : !sv.interface<@handshake_example>
```

Comment:
That type seems to not represent the result.

Reply:
Yeah, it's the input type. The output type can be inferred from it, but not the converse. So I chose to reduce redundancy, though I agree this is probably not intuitive syntax. I'm thinking I'll go with the redundant syntax since it will enable me to use the ODS declarative syntax.